mitosheet/src/mito/components/endo/IndexHeaderContextMenu.tsx

```diff
@@ -50,7 +64,19 @@ export default function IndexHeaderContextMenu(props: {
 
             <DropdownSectionSeperator isDropdownSectionSeperator={true}/>
 
-            <DropdownItem {...getPropsForContextMenuDropdownItem(props.actions.buildTimeActions[ActionEnum.RESET_AND_DROP_INDEX], props.closeOpenEditingPopups)}/>
+            <DropdownItem
+                onMouseEnter={() => setCurrentSubmenu('Copy Row')}
+                subMenu={<Dropdown display={currentSubmenu === 'Copy Row'} position='horizontal' closeDropdown={() => props.setUIState((prevUIState) => {
+                    const isCurrOpenDropdown = isCurrOpenDropdownForCell(prevUIState, props.rowIndex, -1);
+                    return {
+                        ...prevUIState,
+                        currOpenDropdown: isCurrOpenDropdown ? undefined : prevUIState.currOpenDropdown
+                    }
```

Comment:
Is this code still right? I'm a little confused. Maybe this should be closing the subdropdown only?